src/components/Header/index.jsx

```diff
@@ -9,9 +9,9 @@ export default function Header() {
         <h1 className={styles.gradient_text}>Package, dependency and environment management for any language</h1>
         <p>Conda is an open source package management system and environment management system that runs on Windows, macOS, Linux and z/OS. Conda quickly installs, runs and updates packages and their dependencies.</p>
 
-        <div className={styles.header_content_input}>
-          <a href="/docs/intro">Get Started</a>
-        </div>
+      <div class="col col--3 margin--auto margin-top--md">
+        <a class="button button--primary button--lg col" href="/docs/intro">Get Started</a>
```

Comment:
The primary color here refers to Docusaurus' green. I think we should use conda-green1 which is present in custom.css. It will suit more to the theme.

Reply:
Thanks for catching this @Asmit2952.

I think that we should simply update the primary color for the website to be the Conda green. We can do this in this file:

- https://github.com/conda-incubator/conda-dot-org/blob/main/src/css/custom.css

What do you think? That way, we will not need to define additional classes, and I personally prefer the primary color to match the main color used in our designs.

Reply:
Sure, sounds like a good idea. I'll make the necessary changes then